ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/* ext/spl/php_spl.c */
+
+function class_implements($what, bool $autoload = true) : array|false { }
+
+function class_parents($instance, bool $autoload = true) : array|false {}
+
+function class_uses($what, bool $autoload = true) : array|false {}
+
+function spl_autoload(string $class_name, string $file_extensions = '.inc,.php') : void {}
```

Comment:
```suggestion
function spl_autoload(string $class_name, string $file_extensions = UNKNOWN) : void {}
```

Default determined by `spl_autoload_extensions()`.